Dockerfile

```diff
@@ -1,117 +1,155 @@
-
-# Install ruby
-ENV RUBY_VER="2.6.5"
+RUN apt-get update && apt-get --no-install-recommends -y install make autoconf gcc g++ && \
+    cd ~ && \
+    wget https://github.com/jemalloc/jemalloc/archive/${JE_VER}.tar.gz && \
+    tar xf ${JE_VER}.tar.gz && \
+    cd jemalloc-${JE_VER} && \
+    ./autogen.sh && \
+    ./configure --prefix=/opt/jemalloc && \
+    make -j$(nproc) > /dev/null && \
+    make install_bin install_include install_lib
+
+# Install Ruby
+ENV RUBY_VER="2.6.6"
 ENV CPPFLAGS="-I/opt/jemalloc/include"
 ENV LDFLAGS="-L/opt/jemalloc/lib/"
-RUN apt update && \
-	apt -y install build-essential \
-		bison libyaml-dev libgdbm-dev libreadline-dev \
-		libncurses5-dev libffi-dev zlib1g-dev libssl-dev && \
-	cd ~ && \
-	wget https://cache.ruby-lang.org/pub/ruby/${RUBY_VER%.*}/ruby-$RUBY_VER.tar.gz && \
-	tar xf ruby-$RUBY_VER.tar.gz && \
-	cd ruby-$RUBY_VER && \
-	./configure --prefix=/opt/ruby \
-	  --with-jemalloc \
-	  --with-shared \
-	  --disable-install-doc && \
-	ln -s /opt/jemalloc/lib/* /usr/lib/ && \
-	make -j$(nproc) > /dev/null && \
-	make install
-
-ENV PATH="${PATH}:/opt/ruby/bin:/opt/node/bin"
-
+RUN apt-get update && apt-get --no-install-recommends -y install \
+    build-essential \
+    bison \
+    libgdbm-dev \
+    libncurses5-dev \
+    libreadline-dev \
+    libffi-dev \
+    libssl-dev \
+    libyaml-dev \
+    zlib1g-dev 
+RUN cd ~ && \
+    wget https://cache.ruby-lang.org/pub/ruby/${RUBY_VER%.*}/ruby-${RUBY_VER}.tar.gz && \
+    tar xf ruby-${RUBY_VER}.tar.gz && \
+    cd ruby-${RUBY_VER} && \
+    ./configure --prefix=/opt/ruby \
+    --with-jemalloc \
+    --with-shared \
+    --disable-install-doc && \
+    ln -s /opt/jemalloc/lib/* /usr/lib/ && \
+    make -j$(nproc) > /dev/null && \
+    make install
+
+# Update PATH
+ENV PATH="/opt/ruby/bin:/opt/node/bin:${PATH}"
+
+# Install mastodon package managers
 RUN npm install -g yarn && \
-	gem install bundler && \
-	apt update && \
-	apt -y install git libicu-dev libidn11-dev \
-	libpq-dev libprotobuf-dev protobuf-compiler
+    gem install bundler
+
+# Install mastodon OS Package
+RUN apt-get update && apt-get --no-install-recommends -y install \
+    git \
+    libicu-dev \
+    libidn11-dev \
+    libpq-dev \
+    libprotobuf-dev \
+    protobuf-compiler
 
 COPY Gemfile* package.json yarn.lock /opt/mastodon/
 
 RUN cd /opt/mastodon && \
-  bundle config set deployment 'true' && \
-  bundle config set without 'development test' && \
-	bundle install -j$(nproc) && \
-	yarn install --pure-lockfile
+    bundle config set deployment 'true' && \
+    bundle config set without 'development test' && \
+    bundle install -j$(nproc) && \
+    yarn install --pure-lockfile
 
+##-------------------------------------------------
 FROM ubuntu:18.04
 
+# Use bash for the SHELL
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+RUN echo "Etc/UTC" > /etc/localtime && \
+    apt-get update && apt-get --no-install-recommends -y install \
+    apt-utils \
```

Comment:
Why do we need this? AFAIK it only speeds up apt install.

Reply:
Do you mean why do we need apt-utils?